### PR TITLE
Add reference option to the decoder to output the reference text to html.

### PR DESCRIPTION
**Pull request by MorinoseiMorizo**
Now decoder can output the reference text to html.

### CODE REVIEW

File: src/include/nmtkit/formatter.h [hunk 1]
```diff
@@ -40,6 +40,12 @@ class Formatter {
   //   os: Target output stream.
   virtual void finalize(std::ostream * os) = 0;
 
+  // Set the reference text file path.
+  //
+  // Arguments:
+  //   ref_file_path: Path to reference text file.
+  virtual void setReferencePath(std::string ref_file_path) = 0;
```
Comment (odashi) [inline src/include/nmtkit/formatter.h]: const std::string

File: src/include/nmtkit/formatter.h [hunk 2]
```diff
@@ -40,6 +40,9 @@ class Formatter {
   //   os: Target output stream.
   virtual void finalize(std::ostream * os) = 0;
 
+  // TODO write explanation
+  virtual void setReferencePath(std::string ref_file_path) = 0;
```
Comment (odashi) [inline src/include/nmtkit/formatter.h]: This looks a HTMLFormatter-specific option and should not be a general function of Formatter for now.

**Comment by odashi** [comment]
Is this option used by some interface (e.g. option of the decoder)?

**Comment by MorinoseiMorizo** [comment]
Oops, I forgot to add decoder.cc. 
I will add it on Monday.

**Comment by MorinoseiMorizo** [comment]
I changed write function on fomatter class.
Now it takes reference line as a argument, and html formatter can make use of it.

**Pull request merged**